puppet/modules/quickstack/manifests/pacemaker/nosql.pp

```diff
@@ -92,7 +92,7 @@
     Exec['all-nosql-nodes-are-up'] ->
 
     quickstack::pacemaker::resource::service {'mongod':
-      options        => 'start timeout=10s',
+      options        => 'op start timeout=10s',
       monitor_params => { 'start-delay' => '10s' },
```

Comment:
We might need to add monitor_params support to ::generic since the scenario here [1] has a slep, which we may want to simulate with a monitor delay of the same increment?

[1] https://github.com/beekhof/osp-ha-deploy/blob/master/pcmk/mongodb.scenario#L54